src/lib/pages/schedules-create.svelte

```diff
@@ -0,0 +1,177 @@
+<script lang="ts">
```

Comment:
This looks highly copy-pasted from schedule-edit, is there any opportunity for more re-use here?

Reply:
Great idea, refactored it into schedule-form-view